modules/xml-sitemap/sitemap-xslt.php

```diff
@@ -0,0 +1,153 @@
+<?php
+/**
+ * XSLT for XML sitemap
+ *
+ * @module xml-sitemap
+ */
+
+$xslt = '<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="2.0"
+				xmlns:html="http://www.w3.org/TR/REC-html40"
+				xmlns:sitemap="http://www.sitemaps.org/schemas/sitemap/0.9"
+				xmlns:news="http://www.google.com/schemas/sitemap-news/0.9"
+				xmlns:image="http://www.google.com/schemas/sitemap-image/1.1"
+				xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+	<xsl:output method="html" version="1.0" encoding="UTF-8" indent="yes"/>
+	<xsl:template match="/">
+		<html xmlns="http://www.w3.org/1999/xhtml">
+			<head>
+				<title>' . esc_html__( 'XML Sitemap', 'jetpack' ) . '</title>
+				<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+				<style type="text/css">
+					body {
+						font: 14px "Open Sans", Helvetica, Arial, sans-serif;
+						margin: 0;
+					}
+
+					a {
+						color: #3498db;
+						text-decoration: none;
+					}
+
+					h1 {
+						margin: 0;
+					}
+
+					#description {
+						background-color: #81a844;
+						color: #FFF;
+						padding: 30px 30px 20px;
+					}
+
+					#description a {
+						color: #fff;
+					}
+
+					#content {
+						padding: 10px 30px 30px;
+						background: #fff;
+					}
+
+					a:hover {
+						border-bottom: 1px solid;
+					}
+
+					th, td {
+						font-size: 12px;
+					}
+
+					th {
+						text-align: left;
+						border-bottom: 1px solid #ccc;
+					}
+
+					th, td {
+						padding: 10px 15px;
+					}
+
+					.odd {
+						background-color: #E7F1D4;
+					}
+
+					#footer {
+						margin: 20px 30px;
+						font-size: 12px;
+						color: #999;
+					}
+
+					#footer a {
+						color: inherit;
+					}
+
+					#description a, #footer a {
+						border-bottom: 1px solid;
+					}
+					#description a:hover, #footer a:hover {
+						border-bottom: none;
+					}
+				</style>
+			</head>
+			<body>
+				<div id="description">
+					<h1>' . esc_html__( 'XML Sitemap', 'jetpack' ) . '</h1>
+					<p>
+						This is an XML Sitemap generated by <a href="http://jetpack.me/">Jetpack</a> plugin, meant to be consumed by search engines like <a href="https://www.google.com">Google</a> or <a href="https://www.bing.com/">Bing</a>.

```

Comment:
Sure. I left this for last since I was focused on making the xml load the xsl correctly when using default permalinks. I've made the strings translatable now.